contrib/completion/bash/docker

```diff
@@ -1788,17 +1788,17 @@ _docker_run() {
 			;;
 		--security-opt)
```

Comment:
note: to be accurate: `:` us also in `$COMP_WORDBREAKS` but there is speacial treatment for it in 

    _get_comp_words_by_ref -n : cur prev words cword

Fix created: #21584